installing/installing_restricted_networks/installing-restricted-networks-preparations.adoc

```diff
@@ -5,38 +5,93 @@ include::modules/common-attributes.adoc[]
 
 [IMPORTANT]
 ====
-You must have access to the internet to obtain the data that populates the mirror
-repository. In this procedure, you place the mirror registry on a bastion host
+You must have access to the internet to obtain the necessary container images.
+In this procedure, you place the mirror registry on a mirror host
 that has access to both your network and the internet. If you do not have access
-to a bastion host, use the method that best fits your restrictions to bring the
-contents of the mirror registry into your restricted network.
+to a mirror host, use the disconnected procedure to copy images to a device you
+can move across network boundaries with.
 ====
 
 include::modules/installation-about-mirror-registry.adoc[leveloffset=+1]
 
 [id="installing-preparing-bastion"]
-== Preparing the bastion host
+== Preparing your mirror host
 
-Before you create the mirror registry, you must prepare the bastion host.
+Before you perform the mirror procedure, you must prepare the host to retrieve content
+and push it to the remote location.
 
 include::modules/cli-installing-cli.adoc[leveloffset=+2]
 
-include::modules/installation-creating-mirror-registry.adoc[leveloffset=+1]
+include::modules/installation-adding-mirror-registry-pull-secret.adoc[leveloffset=+1]
 
-include::modules/installation-local-registry-pull-secret.adoc[leveloffset=+1]
+[IMPORTANT]
+===
+This step requires that you have write access to a container image repository on the
+mirror registry and adding those credentials to a registry pull secret. This secret
+should NOT be used to install a cluster as it would give machines within the cluster
+write access to your mirror registry.
+===
+
+[id="installing-performing-connected-mirror"]
+== Performing a mirror while connected to the internet
 
-//include::modules/installation-adding-registry-pull-secret.adoc[leveloffset=+1]
+$ oc adm release mirror OPENSHIFT_VERSION --to MIRROR_REPOSITORY
 
 include::modules/installation-mirror-repository.adoc[leveloffset=+1]
 
 include::modules/installation-restricted-network-samples.adoc[leveloffset=+1]
 
+[id="installing-performing-disconnected-mirror"]
+== Performing a mirror to disk for use in disconnected environments
+
+When mirroring to disk you will be downloading images as files, moving your
+mirror host (which may be a laptop) or a removable media device (such as a
+USB drive) into the restricted network, and then completing the mirror
+procedure.
+
+$ oc adm release mirror OPENSHIFT_VERSION --to MIRROR_REPOSITORY --dry-run
+# copy the ICS/ICSP
+
+$ mkdir MIRROR_DIR
+$ cd MIRROR_DIR
+$ oc adm release mirror OPENSHIFT_VERSION --to file://openshift/release
+
+# disconnect, move across firewall
+
+$ oc adm release mirror file://openshift/release:OPENSHIFT_VERSION --to MIRROR_REPOSITORY
```

Comment:
Per https://bugzilla.redhat.com/show_bug.cgi?id=1806779#c11, we should update this command via 'oc image mirror', which is being suggested on the output when mirror release image to disk file.